rye/src/cli/init.rs

```diff
@@ -12,7 +12,7 @@ use serde::Serialize;
 use crate::config::{get_default_author, get_latest_cpython, get_python_version_from_pyenv_pin};
 use crate::utils::is_inside_git_work_tree;
 
-const DEFAULT_LOWER_BOUND_PYTHON: &str = "3.8";
+const DEFAULT_LOWER_BOUND_PYTHON: &str = "3.7";
```

Comment:
Actually it is still has lots of usage, it's min ver [indygreg's Python builds](https://github.com/indygreg/python-build-standalone) and still not reached EOL!